packages/libs/lambda-at-edge/src/s3/s3BucketNameFromEventRequest.ts

```diff
@@ -2,5 +2,5 @@ export const s3BucketNameFromEventRequest = (
   request: AWSLambda.CloudFrontRequest
 ): string | undefined => {
   const { region, domainName } = request.origin?.s3 || {};
-  return domainName?.replace(`.s3.${region}.amazonaws.com`, "");
+  return domainName?.includes(region) ? domainName.replace(`.s3.${region}.amazonaws.com`, "") : domainName.replace(`.s3.amazonaws.com`, "");
```

Comment:
Codacy has a fix for the issue: [Replace `·?·domainName.replace(`.s3.${region}.amazonaws.com`,·"")` with `⏎····?·domainName.replace(`.s3.${region}.amazonaws.com`,·"")⏎···`](https://app.codacy.com/gh/serverless-nextjs/serverless-next.js/pullRequest?prid=7719208)
```suggestion
  return domainName?.includes(region)
```

Reply:
I believe this is the automated Codacy suggestion (though it doesn't seem correct?), though the code build itself was also failing due to type errors, so that might be why.